framework/src/utils/RankTwoTensor.C

```diff
@@ -929,29 +929,20 @@ RankTwoTensorTempl<T>::symmetricEigenvaluesEigenvectors(std::vector<T> & eigvals
 
 template <typename T>
 RankTwoTensorTempl<T>
-RankTwoTensorTempl<T>::rowPermutationTensor(std::vector<unsigned int> old_row,
-                                            std::vector<unsigned int> new_row) const
+RankTwoTensorTempl<T>::permutationTensor(std::vector<unsigned int> old_elements,
```

Comment:
can you pass in a const reference instead, please

Reply:
maybe you can use a `TypeVector<unsigned int>` here? Or an `std::array<unsigned int, LIBMESH_DIM>`

Reply:
I think TypeVector's constructor is private here, I'll change them to `const std::array<unsigned int, LIBMESH_DIM> &`